pyOCD/flash/__init__.py

```diff
@@ -43,6 +43,7 @@
          'lpc800': Flash_lpc800,
          'lpc11u24': Flash_lpc11u24,
          'lpc1768':  Flash_lpc1768,
+         'lpc4088':  Flash_cortex_m,
```

Comment:
Can you use the lpc1768's flash algorithm?  The one you have selected now will not allow you to program the flash.